Cargo.lock

```diff
@@ -3688,7 +3688,7 @@ dependencies = [
 
 [[package]]
 name = "forest-daemon"
-version = "0.7.2"
+version = "0.8.0"
```

Comment:
This should have been part of `0.8.0` release I guess

Reply:
yeah